NEWS.md

```diff
@@ -32,7 +32,7 @@ Contributions from: Major Hayden, Tom Gundersen
   * Composer now correctly passes UID and GID to org.osbuild.{users,groups}
     stages as ints instead of strings.
 
-  * The subpackages (worker, tests and rcm) now require a matching version of
+  * The subpackages (worker and tests) now require a matching version of
```

Comment:
I would rather not change the NEWS. It just feels weird to me. I do understand that RCM API was never properly supported but still it's just weird.

Reply:
I agree, that was unintentional, I'll revert that part

Reply:
Done.